src/Stormpath/Cache/Cacheable.php

```diff
@@ -0,0 +1,30 @@
+<?php namespace Stormpath\Cache;
+
+abstract class Cacheable {
+
+    protected function resourceIsCacheable($resource)
+    {
+        $cache = true;
+
+        // Check to see if it is a collection
+        // All collections will have an items array in the data
+        // If it is a collection, we do not want to cache it;
+        if(isset($resource->items)) $cache = false;
```

Comment:
I thought we said that all resources had a href.  Not sure what you mean by resources that do not have regions